### PR TITLE
feat(w3c/headers): links 'kinds of documents' for W3C documents

### PR DESCRIPTION
**Pull request by marcoscaceres**
Links to a new W3C document that describes what spec status actually are...

### CODE REVIEW

File: tests/spec/w3c/headers-spec.js [hunk 1]
```diff
@@ -11,20 +11,43 @@ import {
...
+      );
+      expect(w3cLink).withContext(`specStatus: ${specStatus}`).toBeTruthy();
+    }
+
+    for (const specStatus of ["unofficial", "base"]) {
+      const doc = await makeRSDoc(makeStandardOps({ specStatus }));
+      const w3cLink = doc.querySelector(
+        ".head a[href='https://www.w3.org/standards/types']"
+      );
+      expect(w3cLink).toBeNull();
```
Comment (sidvishnoi) [inline tests/spec/w3c/headers-spec.js:47]: With context here.

File: tests/spec/w3c/headers-spec.js [hunk 2]
```diff
@@ -11,20 +11,43 @@ import {
...
-  function contains(el, query, string) {
-    return [...el.querySelectorAll(query)].filter(child =>
-      collapsedTextContent(child).includes(string)
-    );
-  }
+
+  it("links to the 'kinds of documents' only for W3C documents", async () => {
+    const statuses = ["FPWD", "LCWD", "WD", "CR", "CRD", "PR", "REC", "NOTE"];
+    for (const specStatus of statuses) {
+      const w3cDoc = await makeRSDoc(makeStandardOps({ specStatus }));
```
Comment (sidvishnoi) [inline tests/spec/w3c/headers-spec.js:35]: Let's call it `doc` for consistent.

File: src/w3c/templates/headers.js [hunk 3]
```diff
@@ -280,6 +270,22 @@ export default (conf, options) => {
   </div>`;
 };
 
+function renderSpecTitle(conf) {
+  const specType = conf.isCR ? `${conf.longStatus}` : `${conf.textStatus}`;
```
Comment (sidvishnoi) [inline src/w3c/templates/headers.js:274]: SidLint: Unnecessary use of template literals

File: src/w3c/templates/headers.js [hunk 4]
```diff
@@ -280,6 +270,22 @@ export default (conf, options) => {
...
+  const preamble = conf.prependW3C
+    ? html`<a href="https://www.w3.org/standards/types">W3C ${specType}</a
+        >${" "}`
```
Comment (sidvishnoi) [inline src/w3c/templates/headers.js:277]: Maybe add the empty space when it's used, instead of on preamble

File: src/w3c/templates/headers.js [hunk 5]
```diff
@@ -280,6 +270,22 @@ export default (conf, options) => {
   </div>`;
 };
 
+function renderSpecTitle(conf) {
+  const specType = conf.isCR ? `${conf.longStatus}` : `${conf.textStatus}`;
+  const preamble = conf.prependW3C
+    ? html`<a href="https://www.w3.org/standards/types">W3C ${specType}</a
```
Comment (sidvishnoi) [inline src/w3c/templates/headers.js:276]: Is there a subpath or fragment for specific types?

Comment (marcoscaceres) [reply src/w3c/templates/headers.js:276]: There is, but not for the "MVP"... will add that later.

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**